test/token/erc1155/ImmutableERC1155.t.sol

```diff
@@ -5,15 +5,16 @@ import "forge-std/Test.sol";
 import {ImmutableERC1155} from "../../../contracts/token/erc1155/preset/draft-ImmutableERC1155.sol";
 import {IImmutableERC1155Errors} from "../../../contracts/errors/Errors.sol";
 import {OperatorAllowlistEnforcementErrors} from "../../../contracts/errors/Errors.sol";
-import {OperatorAllowlist} from "../../../contracts/allowlist/OperatorAllowlist.sol";
+import {OperatorAllowlistUpgradeable} from "../../../contracts/allowlist/OperatorAllowlistUpgradeable.sol";
```

Comment:
This implies that we are no longer testing OperatorAllowlist, and only testing the Upgradeable version. ???

Reply:
Yes, the plan is to no longer use and test OperatorAllowlist but it is a huge effort to rewrite all the tests for it in forge since ethers ^6.6.0(which we need to help test proxy deploys) will break a lot of the hardhat tests

Reply:
The basic functionalities that are common between OperatorAllowlist and OperatorAllowlistUpgradeable are tested by existing hardhat tests